### ISSUE TITLE
Answer Button Accessibility and interactions

### ISSUE THREAD

**Issue by SolivanLau** [not-so-trivial-pursuit/project04#22]
**Accessibility**

- for keyboard users, answers are not focusable. The tabIndex htm could helpl:
    -  tabIndex{0} on items you want the user to be able to tab/focus into - browser takes care of the order
    -  tabIndex{-1} for items you do not want focus on
  
**Interactivity**

- hover/foucus-visible states for buttons would help users understand that buttons are interactive. The following css properties could help:
    - cursor: pointer
    - background-Color: different color

**UI/UX**
- it was a little difficult understanding that answers were being evaluated on click
- after user clicks on an answer button, it was a little confusing as to why we could change our answer with the correct answer showing
- perhaps on click, you can disable buttons to lock in using a state and turnery expression (ex. questionAnswered ? disabled : null;
